### PR TITLE
Collapsible resizers 

### PR DESCRIPTION
**Pull request by tofarr**
Support for collapsible panels in the UI
image

Separators are still draggable, but users can now click on the Icons to expand / collapse panels:
image

image

image

### CODE REVIEW

**Comment by neubig** [comment]
@amanape , could you take a look? If you're busy I'll grab it though.

File: frontend/src/components/Resizable.tsx [hunk 1]
```diff
@@ -67,33 +90,104 @@ export function Container({
...
-    return { height: `${firstSize}px` };
   };
 
   return (
     <div
-      className={twMerge(
-        `flex ${orientation === Orientation.HORIZONTAL ? "" : "flex-col"}`,
-        className,
-      )}
+      className={twMerge(`flex ${isHorizontal ? "" : "flex-col"}`, className)}
```
Comment (amanape) [inline frontend/src/components/Resizable.tsx:160]:
```suggestion
      className={twMerge("flex", isHorizontal && "flex-col", className)}
```

**Review by amanape — APPROVED** [review]
LGTM! Though you may consider wrapping some of the handlers in a `useCallback` since this component is re-render-intensive

**Comment by amanape** [comment]
@tofarr Does this resolve #3199 / #1745?

**Comment by tofarr** [comment]
> @tofarr Does this resolve #3199 / #1745?

I think so - panels are now resizable and no longer have a min size. Though on of the changes since deploy seems to have done some thing odd

**Comment by tofarr** [comment]
The fix to isHorizontal inadvertently inverted the logic, breaking the layout - so I added one more fix:
Broken:
image

Fixed:
image

**Pull request merged**